cmd/neofs-cli/modules/control/shards_set_mode.go

```diff
@@ -36,7 +37,8 @@ func initControlSetShardModeCmd() {
 	flags := setShardModeCmd.Flags()
 
 	flags.String(controlRPC, controlRPCDefault, controlRPCUsage)
-	flags.String(shardIDFlag, "", "ID of the shard in base58 encoding")
+	flags.StringSlice(shardIDFlag, nil, "List of shard IDs in base58 encoding")
```

Comment:
is that a compatible change? (i think yes, just to be sure)

Reply:
Yes, old commands will continue to work